com.maplerain.telsis_translator_flutter.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=Telsis language translator
+Comment=App that translates to and from the Telsis language used in the Violet Evergarden series
+Categories=Utility;Amusement;
+
+Icon=com.maplerain.telsis_translator_flutter
```

Comment:
Same about the icon file.

Reply:
Yes, it is in the tarball. I saw this file on other repos and blindly followed. My apologies. 🙇‍♀️